src/client/components/pages/privacy.js

```diff
@@ -20,6 +20,11 @@ const React = require('react');
 
 const PageHeader = require('react-bootstrap').PageHeader;
 
+/**
+* Description of the function PrivacyPage
+* the privacy policy of MusicBrainz
```

Comment:
MusicBrainz? While it does link to the privacy policy of MusicBrainz, your description here is not what I would take away from the function.